rfc/0010-cbft.md

```diff
@@ -96,7 +96,16 @@ This RFC describes the SDK API and behavior exposure of the Couchbase Server Ful
 			- [Numeric Range Facet Results](#numeric-range-facet-results)
 			- [Date Range Facet Results](#date-range-facet-results)
 - [Language Specifics](#language-specifics)
-- [Unresolved Questions](#unresolved-questions)
+		- [Set-Up for Examples](#set-up-for-examples)
+		- [Java](#java)
+		- [.NET](#net)
+		- [NodeJS](#nodejs)
+		- [Go](#go)
+		- [C](#c)
+		- [PHP](#php)
+		- [Python](#python)
+		- [Ruby](#ruby)
+- [Resolved Questions](#resolved-questions)
```

Comment:
Need examples now

- [ ] .NET, need one from @jeffrymorris 
- [ ] Node, need one from @brett19 
- [ ] Go, need one from @brett19 
- [ ] C, need an example from @mnunberg 
- [ ] PHP, need an example from @avsej 
- [ ] Python, need an example from @mnunberg 

We can remove the Ruby one.